### PR TITLE
✨ [Fathom Analytics] Release v1 of Fathom Analytics plugin

### PR DESCRIPTION
**Pull request by DanielFerguson**
## Description

Added the [Fathom Analytics](https://usefathom.com) Raycast extension.

Quickly interact with your Fathom Analytics account straight from your desktop. Quick open to a site dashboard, create a site and retrieve a site code to install on your new project, delete a site and open to your main dashboard; all with the power and efficiency you've come to love from Raycast.

## Screencast

[Fathom Analytics Raycast Demo Video](https://www.youtube.com/watch?v=MWIztNmsCR4)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by DanielFerguson** [comment]
@danielsequeira Pushed the fathom analytics plugin without the Typefully changes. 🙌

**Review by danielsequeira — COMMENTED** [review]
Looks good @DanielFerguson, just a tiny comment from my side 🙂

File: extensions/fathom-analytics/package.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      "title": "Open Dashboard",
+      "subtitle": "Fathom Analytics",
+      "description": "Open Fathom Analytics.",
+      "mode": "no-view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "apiToken",
+      "description": "Enter your API token from Fathom Analytics.",
```
[the remark this answers is not in the document]
Comment (DanielFerguson) [reply extensions/fathom-analytics/package.json]: Donezo! 🥳 I'll try and remember this CX style for future extensions as well.

**Comment by DanielFerguson** [comment]
Done, @danielsequeira! Thank you for the suggestion - I will try and remember this CX style for future plugins and extensions. 😊

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**